.github/workflows/koncur-test.yaml

```
@@ -0,0 +1,82 @@
+      koncur_ref:
+        type: string
+        default: main
+        description: Branch or tag of koncur repository to test against
+
+jobs:
+  koncur-test:
+    name: Set up koncur tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout kantra
+        uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)